frontend/controller/service-worker.js

```diff
@@ -204,20 +206,24 @@ sbp('sbp/selectors/register', {
         ? await registration.pushManager.getSubscription().then((subscription) => {
           if (
             !subscription ||
-            (subscription.expirationTime != null &&
-            subscription.expirationTime <= Date.now())
+            (subscription.expirationTime != null && subscription.expirationTime <= Date.now())
           ) {
-            console.info(
-              'Attempting to create a new subscription',
-              subscription
-            )
+            console.info('[service-worker/setup-push-subscription] attempting to create a new subscription', subscription)
             return sbp('push/getSubscriptionOptions').then(function (options) {
               return registration.pushManager.subscribe(options)
             })
+          } else {
+            const subClone = cloneDeep(subscription)
+            if (subscription?.endpoint) {
+              // hide the full endpoint from the logs for privacy
+              subClone.endpoint = new URL(subClone.endpoint).host
```

Comment:
Not sure if this helps much, as the host will likely be the same for all browsers of the same vendor.

Reply:
No, I get the privacy angle (I don't think there's something _stopping_ a push hostname to contain sensitive information, but in practice they don't). What I'm saying is that it makes logging it almost worthless, as you can predict with high confidence what it'll be based on the user agent.